src/components/Filters/CheckboxOption.tsx

```diff
@@ -118,7 +118,7 @@ export function CheckboxOption(props: CheckboxOptionProps): JSX.Element | null {
 
   const isSelected = existingStoredFilter ? existingStoredFilter.selected : false;
 
-  const labelText = resultsCount ? `${displayName} (${resultsCount})` : displayName;
+  const labelText = (resultsCount ? `${displayName} (${resultsCount})` : displayName).toString();
```

Comment:
we don't need to call toString on this whole statement. the first part of the ternary already returns a template string. also, if displayName was of type NumberRangeValue, wouldn't this display `[Object object]` or something instead of a useful label?